openrtb_ext/request_wrapper.go

```diff
@@ -1429,3 +1589,31 @@ func (e *ImpExt) GetGpId() string {
 func CreateImpExtForTesting(ext map[string]json.RawMessage, prebid *ExtImpPrebid) ImpExt {
 	return ImpExt{ext: ext, prebid: prebid}
 }
+
+func (e *ImpExt) Clone() *ImpExt {
+	if e == nil {
+		return nil
+	}
+
+	clone := *e
+	clone.ext = maputil.Clone(e.ext)
+
+	if e.prebid != nil {
+		clonedPrebid := *e.prebid
+		clonedPrebid.StoredRequest = ptrutil.Clone(e.prebid.StoredRequest)
+		clonedPrebid.StoredAuctionResponse = ptrutil.Clone(e.prebid.StoredAuctionResponse)
+		if e.prebid.StoredBidResponse != nil {
+			clonedPrebid.StoredBidResponse = make([]ExtStoredBidResponse, len(e.prebid.StoredBidResponse))
+			for i, sbr := range e.prebid.StoredBidResponse {
+				clonedPrebid.StoredBidResponse[i] = sbr
+				clonedPrebid.StoredBidResponse[i].ReplaceImpId = ptrutil.Clone(sbr.ReplaceImpId)
+			}
+		}
+		clonedPrebid.IsRewardedInventory = ptrutil.Clone(e.prebid.IsRewardedInventory)
+		clonedPrebid.Bidder = maputil.Clone(e.prebid.Bidder)
+		clonedPrebid.Options = ptrutil.Clone(e.prebid.Options)
+		clone.prebid = &clonedPrebid
```

Comment:
I think my previous comment on this line might have been missed but looks like we need to copy over `impExt.Prebid.Floors` as well here.